course_flow/static/course_flow/js/PostFunctions.js

```diff
@@ -780,6 +780,20 @@ export function getParentWorkflowInfo(workflowPk,callBackFunction=()=>console.lo
     }
 }
 
+//Get the public data from the workflow
+export function getPublicParentWorkflowInfo(workflowPk,callBackFunction=()=>console.log("success")){
+    try{
+        $.get(

```

Comment:
Is there a timeline for phasing out jQuery?  Not critical, but we are going the other way in dalite... no new code with jQuery and removing old code that depends on it over time.

Reply:
I really like jQuery, so unless there's a very compelling reason to phase it out I would not do so.